Introduction/README.md

```diff
@@ -415,8 +480,8 @@ The result is once again the same.
 
 ## Wrap Up
 
-Congratulations! In this tutorial you've accomplished a lot. You've created your
-first function, deployed it to your local Fn server and invoked it
-over HTTP.
+Congratulations!  In this tutorial you've accomplished a lot.  You've created
+your first function, deployed it to your local Fn server and invoked it over
+HTTP.
 
-**Go:** [Back to Contents](../README.md)
+**Go:** [Back to Contents](../../README.md)
```

Comment:
@michael-w-williams going two levels up lands us with a 404 on Github, since it's out of the project tree. I'll modify this back to one level up before merging. Hope you're ok with that.